src/component/clientPatientRegistration/ClientPatientRegisterPage.tsx

```diff
@@ -97,24 +97,40 @@ console.log(registerInfo);
     value: website,
   }));
 
+  const initialvalue =
+    registerInfo.firstName == "defaultfromslice"
+      ? {}
+      : {
+          firstName: registerInfo.firstName,
+          lastName: registerInfo.secondName,
+          birthDate: stringToDate(registerInfo.birthDate),
+          ethnicity: registerInfo.ethnicity,
+          gender: registerInfo.gender,
+          email: registerInfo.email,
+          address: {
+            state: registerInfo.address.state,
+            city: registerInfo.address.city,
+            street: registerInfo.address.street,
+          },
+          paymentMethod: registerInfo.paymentMethod,
+          insuranceProvider: registerInfo.insuranceProvider,
+        };
+
   return (
     <Form
       {...formItemLayout}
       form={form}
       name="register"
       onFinish={onFinish}
-      initialValues={{
-        residence: ["zhejiang", "hangzhou", "xihu"],
-        prefix: "86",
-      }}
+      initialValues={initialvalue}
       scrollToFirstError
     >
       <Form.Item
         name="firstName"
-        label="First Name"
+        label="First Namewww"
         rules={[
           {
-            required: true,
+            required: false,
```

Comment:
should not at least some of the fields be required? Forms can be submitted without any input if every field is kept optional.

Reply:
This was done for debugging purpose, as filling all the forms or at least one label everytime was time consuming.